java_gen/pre-written/src/test/java/org/projectfloodlight/openflow/types/MacAddressTest.java

```diff
@@ -151,4 +151,38 @@ public void testIsLLDPAddress() {
         assertFalse(MacAddress.of("00:80:C2:f0:00:00").isLLDPAddress());
         assertFalse(MacAddress.of("FE:80:C2:00:00:00").isLLDPAddress());
     }
+
+    @Test
+    public void generateMac()
```

Comment:
Maybe call this `testForIPv4MulticastAddress()` so that it's more obvious what the test is for?